skrf/tests/test_network.py

```diff
@@ -55,6 +55,31 @@ def test_timedomain(self):
         t = self.ntwk1.s11.s_time
         self.assertTrue(npy.sum(npy.abs(t.imag)) == 0)
 
+    def test_time_transform(self):
+        spb = (4, 5)
+        data_rate = 5e9
+        num_taps = (100, 101)
+        for i in range(2):
+            tps = 1 / spb[i] / data_rate
+            num_points = spb[i] * num_taps[i]
+            # Frequency terms should NOT contain Nyquist frequency if number of points is odd
+            inc_nyq = True if num_points % 2 == 0 else False
+            freq = npy.linspace(0, 1 / 2 / tps, num_points // 2 + 1, endpoint=inc_nyq)
```

Comment:
thanks for making this test. 
1/2 evaluates to 0 in py2.7.  i think you need to use 1.0/2 or similar for force float division.  you can see the results in travis https://travis-ci.org/scikit-rf/scikit-rf/jobs/340844075